src/tests/roc_lib/test_sender_receiver.cpp

```diff
@@ -212,29 +215,40 @@ class Receiver {
             }
 
             if (!seek_first) {
-                for (; i < frame_size_; i++) {
+                for (; i < frame_size_; i++, sample_num++, middle_stream_total++) {
                     if (sample_num >= total_samples_) {
                         roc_panic_if_not(is_zero_(rx_buff[i]));
                         finish = true;
                         roc_log(LogDebug, "finish: leading_zeros: %lu, num_samples: %lu",
                                 (unsigned long)leading_zeros, (unsigned long)sample_num);
                         break;
-                    } else if (!is_zero_(samples_[sample_num] - rx_buff[i])) {
+                    } 
+                    
+                    if (is_zero_(samples_[sample_num] - rx_buff[i])) {
+                        middle_stream_identical++;
+                    } else if (!is_zero_(rx_buff[i])) {
                         char sbuff[256];
-                        int sbuff_i =
-                            snprintf(sbuff, sizeof(sbuff),
-                                     "failed comparing sample #%lu\n\nframe_num: %lu\n",
-                                     (unsigned long)sample_num, (unsigned long)frame_num);
+                        int sbuff_i = snprintf(sbuff, sizeof(sbuff),
+                            "failed comparing sample #%lu\n\nframe_num: %lu\n",
+                            (unsigned long)sample_num, (unsigned long)frame_num);
                         snprintf(&sbuff[sbuff_i], sizeof(sbuff) - (size_t)sbuff_i,
-                                 "original: %f,\treceived: %f\n",
-                                 (double)samples_[sample_num], (double)rx_buff[i]);
+                            "original: %f,\treceived: %f\n",
+                            (double)samples_[sample_num], (double)rx_buff[i]);
                         roc_panic("%s", sbuff);
-                    } else {
-                        sample_num++;
                     }
                 }
             }
         }
+
+        double identical = (double)middle_stream_identical / middle_stream_total;          
+        if(identical < .9) {
```

Comment:
@alexandremgo could you please run `scons -Q fmt` to use the existing formatting style?

Reply:
yes sorry